sdk/dotnet/Pulumi/Deployment/TaskMonitoringHelper.cs

```diff
@@ -0,0 +1,191 @@
+// Copyright 2016-2021, Pulumi Corporation
+
+using System;
+using System.Linq;
+using System.Collections.Generic;
+using System.Collections.Immutable;
+using System.Threading;
+using System.Threading.Tasks;
+
+namespace Pulumi
+{
+    /// Monitors dynamically added tasks for completion. Enters IDLE
```

Comment:
It's not public API but anyway the convention is to wrap the comments between two

```
/// <summary>
/// </summary>
```

I believe we do so fairly consistently.